src/helpers/__tests__/fs.test.ts

```diff
@@ -0,0 +1,218 @@
+import fs from 'fs'
+import path from 'path'
+import process from 'process'
+
+import JSZip from 'jszip'
+
+import {createDirectories, deleteFolder, writeFile, zipContents} from '../fs'
+
+import {MOCK_DATADOG_API_KEY} from './fixtures'
+
+// Mock constants
+const MOCK_CWD = 'mock-cwd'
+const MOCK_FOLDER_NAME = 'mock-folder'
+const MOCK_FOLDER_PATH = path.join(MOCK_CWD, MOCK_FOLDER_NAME)
+const MOCK_FILE_NAME = 'function_config.json'
+const MOCK_ZIP_PATH = 'output.zip'
+const MOCK_FILES = new Set([MOCK_FILE_NAME, 'file1.csv', 'file2.csv', 'file3.csv'])
+const MOCK_CONFIG = {
```

Comment:
```suggestion
const MOCK_AWS_LAMBDA_CONFIG = {
```
or `MOCK_LAMBDA_CONFIG`